.github/workflows/test.yml

```diff
@@ -952,8 +948,7 @@ jobs:
             './tfm/cypress/e2e/journeys/case-amendments/amendment-should-change-all-facilities-values/*.spec.js',
             './tfm/cypress/e2e/journeys/case-amendments/amendment-underwriting-page-multiple-amendments/*.spec.js',
             './tfm/cypress/e2e/journeys/case-amendments/amendments-version-order/*.spec.js',
-            './tfm/cypress/e2e/journeys/case-amendments/facility-end-date/bss-deal-expect-no-facility-end-date-features/*.spec.js',
-            './tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-enabled/*.spec.js',
+            './tfm/cypress/e2e/journeys/case-amendments/facility-end-date/**.spec.js',
```

Comment:
I think this should be `facility-end-date/*.spec.js`, unless there are subfolders, then it should be `**/*.spec.js`

Reply:
Have updated :)